conda/cli/conda_argparse.py

```diff
@@ -267,9 +267,9 @@ def configure_parser_info(sub_parsers):
         help="Display information about packages.",
     )
     p.add_argument(
-        '--root',
+        '--base',
         action='store_true',
-        help='Display root environment path.',
+        help='Display base environment path.',
     )
     p.add_argument(
```

Comment:
We need to keep root here for backward compatibility.  So, keep `--base` here as you've modified it.  But then also add

```python
p.add_argument(
    '--root',
    action='store_true',
    help=SUPRESS,
    dest='base',
)
```